src/psij/testing/service.py

```diff
@@ -450,7 +453,9 @@ def getSite(self, site_id, tests_to_match):
                     if testName == test:
                         foundMatch = 1
 
-                if oneTest['branch'] == 'main' and foundMatch == 1:
+                branchMatch = oneTest['branch'] == 'master'
+
```

Comment:
I might not understand things quite right, but aren't these tests redundant? We're already filtering both the branches and test names, so I'm not sure how checking that they are really matching the filter helps.